lib/make_call.rb

```diff
@@ -0,0 +1,19 @@
+require 'twilio-ruby'
+require_relative 'order'
+class Checkout
+
+  def initialize
+    account_sid = 'ACdb0c001d98d842c9465935ffde45fb6b'
+    auth_token = 'aa648095dec33b70637f15de4c6c9964'
+    @client = Twilio::REST::Client.new account_sid, auth_token
+    @order = Order.new
```

Comment:
This does not belong here - the Checkout is not responsible for creating an order.